ch-06/src/Solution632.java

```diff
@@ -0,0 +1,69 @@
+public class Solution632 {
+    public static void main(String[] args) {
+        final int NUMBER_OF_GAMES = 15000;
+        int losingCounter = 0;
+        int winningCounter = 0;
+
+        for (int i = 1; i <= NUMBER_OF_GAMES; i++) {
+            int firstDiceInFirstRoll = rollTheDice();
+            int secondDiceInFirstRoll = rollTheDice();
+            final int sumOfFirstRoll = firstDiceInFirstRoll + secondDiceInFirstRoll;
+
+            if (isThePlayerWinningAfterFirstRoll(sumOfFirstRoll)) {

```

Comment:
An immediate thought would be, for someone unfamiliar with the implementation and how the game works, is to refactor it somehow like this:

~~~~
if (isThePlayerWinningAfterFirstRoll || isWinningInSecondRoll) {
  winningCounter++;
} else {
  losingCounter++;
}
~~~~

However, that's, of course, wrong. So, in this case, comments would really help for readers to understand the dependencies between the `else if` arms.